### ISSUE TITLE
Blank Space inside Matterfront window

### ISSUE THREAD

**Issue by asadpiz** [lloeki/matterfront#84]
Hi I am trying to find a simple front-end windows desktop app for mattermost and while I follow the steps mentioned in MatterFront (README.md). When I launch the app (**npm run start**) I just get a Window titled "Matterfront" and there is nothing inside the window (just blank space).

I am doing nothing fancy, I have a VM with a test team [<url>/testteam] and in my config.json file I just add the following text:

```

{
  "teams": [{
    "url": "http://<url>:8065/testteam"
  }]
}
```

My system is Windows 7 x64

![log](https://cloud.githubusercontent.com/assets/3435875/12983719/cceb334e-d0ea-11e5-9f3f-5c29c6324c2d.PNG)

*_Command Prompt:
*_![mf](https://cloud.githubusercontent.com/assets/3435875/12983554/0eac927e-d0ea-11e5-953d-2172717e98db.PNG)

PS: I can use the compiled version of this app at: https://github.com/yuya-oc/electron-mattermost
But I need the system tray icon notification feature which isn't available

**Comment by LongLiveCHIEF** [comment]
Is you mattermost on an internal network, or are you behind a proxy?

**Comment by asadpiz** [comment]
Yes the mattermost server is in an internal network and the entire system is behind a proxy BUT:
1. I have also tried running mattermost on a local VM and the app  (when executed from host) gives the same problem.
2. The precompiled app (electron-mattermost) works fine with our main mattermost server (which is in a DMZ).

**Comment by LongLiveCHIEF** [comment]
Please refer to the [command-line args](https://github.com/HackerHappyHour/matterfront/blob/master/README.md#electron-and-chrome-command-line-args) section of the setup instructions.  Please use the `no-proxy-server` option.
